### ISSUE TITLE
Output2.csv on the dictionary branch has october data

### ISSUE THREAD

**Issue by rcackerman** [rcackerman/parole-hearing-data#20]
Just an FYI @nikzei - output2.csv on the dictionary branch is the most up-to-date csv. Once I get the dictionary stuff working I'll merge into master. Please don't merge until then.

**Comment by talos** [comment]
@rcackerman Now that the dictionary branch is merged into master, what's the status of this data? Should it be eliminated? `data.csv` was created mor recently.

**Comment by rcackerman** [comment]
See #27. Closing this as the dictionary branch has now been merged into master.
